### PR TITLE
Change a warning of Firefox about:config entering

### PR DESCRIPTION
**Pull request by igoose1**
## Description

Button had been changed to "Accept the Risk and Continue" so I've updated that. More information at #1602

Resolves: https://github.com/privacytoolsIO/privacytools.io/issues/1532

#### Check List 

- [x] I have read and understand [the contributing guidelines](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- [x] The project is [Free Libre](https://en.wikipedia.org/wiki/Free_software) and/or [Open Source](https://en.wikipedia.org/wiki/Open-source_software) Software

### CODE REVIEW

File: _includes/sections/browser-tweaks.html [hunk 1]
```diff
@@ -6,7 +6,7 @@ <h3>Preparation:</h3>
 
 <ol>
   <li>Enter "about:config" in the firefox address bar and press enter.</li>
-  <li>Press the button "I'll be careful, I promise!"</li>
+  <li>Press the button "Accept the Risk and Continue"</li>
```
Comment (Mikaela) [inline _includes/sections/browser-tweaks.html]: Has it also changed in Firefox ESR?

Comment (igoose1) [reply _includes/sections/browser-tweaks.html]: > Has it also changed in Firefox ESR?

I cannot check that on my system :woman_shrugging:. I'll try to look at someone's ESR version but I cannot promise that. Right now I can say it is, at least, on 71.0, but that's a "standard" version, not ESR.

**Comment by igoose1** [comment]
I added "or" with both wordings but it must be better to check if ESR really hasn't changed the sentence.

**Comment by Thorin-Oakenpants** [comment]
ESR 68.0.2, FF70 and lower (tested back to FF60)
- `I accept the risk`

FF71+
- `Accept the Risk and Continue`

I only checked on Windows, but I doubt the OS has any affect here.

---

No idea where you're getting `I'll be careful, I promise!` from but it looks vaguely familiar - maybe that's an even older wording?

**Comment by Mikaela** [comment]
@igoose1 Would you be open to committing based on Thorin-Oakenpants's comment?

**Comment by igoose1** [comment]
@Mikaela, here is. Thank you, @Thorin-Oakenpants!

**Pull request merged**